web-app/packages/lib/src/common/components/FullStorageWarningTemplate.vue

```diff
@@ -31,11 +31,13 @@ import { useUserStore } from '@/main'
 const userStore = useUserStore()
 
 const usage = computed(() =>
-  Math.floor(
-    (userStore.currentWorkspace?.disk_usage /
-      userStore.currentWorkspace?.storage) *
-      100
-  )
+  userStore.currentWorkspace?.disk_usage > userStore.currentWorkspace?.storage
+    ? 0
```

Comment:
should not be 100%?

Reply:
if storage === 0 , this is not possible to make disk_usage / storage. We do not want to show this banner in such edge case.

Reply:
but disk usage can be > storage in other cases, like plan downgrade?

Reply:
yeah, but in these cases -> it is not reasonable to warn user about full storage :)

Reply:
ah, that is only warning, sorry, I missed it completely